docs/Paginator.md

```diff
@@ -0,0 +1,14 @@
+# Paginator
+
+## Props
+| Name             | Type             | Default         | Required | Description                                                                                                                                                                                                                                                   |
+| ---------------- | ---------------- | --------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| onChangePage     | number =&gt; any |                 | true     | callback function triggered when page is changed                                                                                                                                                                                                              |
+| currentPage      | number           |                 | true     | Used to determine whether the disabled props is true or not for the<br>`next page button` and `prev page button`<br>if props `currentPage` < `totalPages` `next page button` is not disabled<br>if props `currentPage > 1` `prev page button` is not disabled |
+| totalPages       | number           |                 | true     | Indicates the total number of pages: number                                                                                                                                                                                                                   |
+| isLoading        | boolean          | false           | false    | If true it will show is `isLoadingContent`                                                                                                                                                                                                                    |
+| isLoadingContent | Function         | () => 'loading' | false    | Function that renderers jsx                                                                                                                                                                                                                                   |
+| isEmptyContent   | Function         | () => 'empty'   | false    |                                                                                                                                                                                                                                                               |
```

Comment:
renders content when result set is empty - i dont think this functionality actullay is used

Reply:
added description but kept logic